internal/util/util_test.go

```diff
@@ -0,0 +1,108 @@
+package util
+
+import (
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	platformv1alpha1 "github.com/openshift/api/platform/v1alpha1"
+	platformtypes "github.com/openshift/platform-operators/api/v1alpha1"
+)
+
+func Test_inspectPlatformOperator(t *testing.T) {
```

Comment:
Is this name formatted this way for a reason? Why not just `TestInspectPlatformOperator`?

Reply:
This was auto-generated for me by vscode. We can refactor down the line if needed.